src/views/v2/rsvp/attendees/attendee.php

```diff
@@ -15,15 +15,18 @@
  * @var array $attendees List of attendees for the given order.
  * @var int $attendee_id The attendee ID.
  *
+ * @since TBD - Add attendee name and rsvp name templates.
  * @since 5.7.0
  *
- * @version 5.7.0
+ * @version TBD
  */
 
 if ( empty( $attendees ) ) {
 	return;
 }
+
 ?>
 <div class="tec-tickets__attendees-list-item-attendee-details">
```

Comment:
We can potentially add the `tribe-common-b1` here so that the elements inside inherits the styles and we don't have to repeat ourselves.